src/src/DataStructs/SettingsStruct.h

```diff
@@ -171,6 +171,8 @@ class SettingsStruct_tmpl
   uint32_t      VariousBits1;
   uint32_t      ResetFactoryDefaultPreference; // Do not clear this one in the clearAll()
   uint32_t      I2C_clockSpeed;
+  uint16_t      WebserverPort;
```

Comment:
You can increase the `BUILD` number and then call `BuildFixes()`

See where that's called:
```c++
  if (Settings.Build != BUILD)
    BuildFixes();
```

Reply:
we got validate() just after load settings so maybe it is not needed?